c/list-ext3-properties/sll_circular_traversal-1.c

```diff
@@ -63,7 +65,7 @@ int main() {
     free(ptr);
     ptr = temp;
```

Comment:
isn't temp also pointing to a freed object?

Reply:
in the last iteration, yes, then the next-pointer refers to a freed object and should not be used. I will try to fix this.
C is really broken here.